### ISSUE TITLE
Dark mode background color should match the mkdocs_material dark mode background color

### ISSUE THREAD

**Issue by smartYSC** [MikhailKravets/mkdocs_puml#70]
**Is your feature request related to a problem? Please describe.**
When a diagram is rendered in light mode, the background color used by `mkdocs_material` is matched exactly (white). You see no border of the diagram. In dark mode, the background color is not matching and the diagram has a visible border.

**Describe the solution you'd like**
There should be a dark mode theme (probably named `material`) that matches the `mkdocs_material` colors.

**Describe alternatives you've considered**
I played around with a few of the themes but did not fine one that matches (yet).

**Comment by MikhailKravets** [comment]
@smartYSC, I intentionally set the appropriate background color in themes. As for [default](https://github.com/MikhailKravets/mkdocs_puml/blob/master/themes/default/README.md) theme, I just copied it from PlantUML with its background settings.

I personally like when diagrams have distinctive frame. It makes them consistent with their color scheme. If we change it on transparent altogether, it can potentially disrupt the look and feel. Additionally, for me it was harder to understand the field which I control.

However, if you want to add transparent background, you may set it explicitly either with skinparam

```plantuml
skinparam backgroundcolor transparent
```

or with new styling API

```plantuml
<style>
document {
    BackGroundColor transparent
}
<style>
```

**Comment by smartYSC** [comment]
OK understood. Would there be interest in a set of `material` themes that closely match the color (and maybe other aspects) of the `default` and `slate` themes of `mkdocs-material`? One thing to do would to set the colors to the primary/accent color of the mkdocs theme.

**Comment by MikhailKravets** [comment]
Yes, that would be nice. However, unlike the other implemented themes where I just used existing color palette, for material we should first develop a color scheme. PlantUML requires much more colors than mkdocs-material currently provide

**Comment by smartYSC** [comment]
I haven't checked that in detail, but apart from the primary color, at least the admonitions provide a lot of existing colors that work well together. So I guess it would be possible to switch to those.

Also setting the font to `Roboto` would probably go a long way already.

**Comment by MikhailKravets** [comment]
I will mark it as a feature request for later implementation. My next priority is to update project documentation.

**Comment by MikhailKravets** [comment]
Hi @smartYSC. Material theme is now available in `2.3.0` version of the plugin... but sadly without material fonts. Despite how much I tried to make PlantUML to use Roboto font, it was constantly using SansSerif :shrug:

**Comment by smartYSC** [comment]
This looks very cool! Thank you, we will definitely use it!
